cmd/chart-repo/utils_test.go

```diff
@@ -238,6 +238,21 @@ func Test_importCharts(t *testing.T) {
 	}
 }
 
+func Test_DeleteRepo(t *testing.T) {
+	m := &mock.Mock{}
+	m.On("Upsert", mock.Anything)
+	m.On("RemoveAll", mock.Anything)
+	dbSession := mockstore.NewMockSession(m)
+	index, _ := parseRepoIndex([]byte(validRepoIndexYAML))
+	charts := chartsFromIndex(index, repo{Name: "test", URL: "http://testrepo.com"})
+	importCharts(dbSession, charts)
+
+	err := deleteRepo(dbSession, "test")
+	if err != nil {
+		t.Errorf("failed to delete chart repo test: %v", err)
+	}
```

Comment:
we should run `m.AssertExpectations` to ensure our `m.On` calls match what actually happens.